.github/workflows/dev-release-grpc-service-pypi-publish.yml

```
@@ -0,0 +1,22 @@
+name: Dev release and publish Python distributions to PyPI. New gRPC generator
+
+on:
+  push:
+    tags:
+      - \d+.\d+.\d+-dev
+
+jobs:
+  collect-version:
+    name: Get version from gradle.properties
+    uses: th2-net/.github/.github/workflows/compound-prebuild-java-dev-workflow.yml@main
+  build-n-publish:
+    name: Dev build and Python distributions to PyPI
+    needs: [collect-version]
+    uses: th2-net/.github/.github/workflows/compaund-python-grpc-pypi-publication.yml@main
+    with:
+      custom-version: ${{ needs.collect-version.outputs.versionNumber }}-dev
```

Please, use `rc1` instead `-dev`. Because `-dev` will be converted to `.dev0` automatically and that is not what we need
```suggestion
      custom-version: ${{ needs.collect-version.outputs.versionNumber }}rc1
```